xml/System.Windows.Forms/SystemInformation.xml

```diff
@@ -4051,9 +3983,7 @@
  The <xref:System.Windows.Forms.SystemInformation.TerminalServerSession%2A> property indicates whether the calling process is associated with a Terminal Services client session.  
   
 > [!NOTE]
->  This property is supported only on Windows NT 4.0 SP4 or later, Windows 98, Windows Millennium Edition, Windows 2000, Windows XP, and the Windows Server 2003 family.  
-  
-   
+> This property is supported only on Windows NT 4.0 SP4 or later, Windows 2000, Windows XP, and the Windows Server 2003 family.
```

Comment:
Do we still support any of these? Or should the whole sentence be removed?

Reply:
This and similar notes must be removed.